UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -145,7 +148,7 @@ public partial class GameManager : MonoBehaviour, IInitialise
 	public bool LoadedDirectlyToStation => loadedDirectlyToStation;
 
 	public bool QuickLoad = false;
-
+	public bool QuickJoinLoad = false;
```

Comment:
why was this renamed?

Reply:
it's a new field doesn't replace the old one